src/dotnet/commands/dotnet-pack/SymbolPackageGenerator.cs

```diff
@@ -26,8 +26,10 @@ protected override void ProcessContext(ProjectContext context)
             base.ProcessContext(context);
 
             var inputFolder = ArtifactPathsCalculator.InputPathForContext(context);
-            TryAddOutputFile(context, inputFolder, $"{Project.Name}.pdb");
-            TryAddOutputFile(context, inputFolder, $"{Project.Name}.mdb");
+            var ouptutName =  Project.GetCompilerOptions(context.TargetFramework, Configuration).OutputName;
```

Comment:
`GetCompilerOptions(context.TargetFramework, Configuration)` is what I was referring to. 

Just wanted to double check so we don't get a null ref exception :smile: